### PR TITLE
fix: remove unused encoding parameter

### PR DESCRIPTION
**Pull request by anonrig**

### CODE REVIEW

File: src/url_aggregator.cpp [hunk 1]
```diff
@@ -99,4 +99,8 @@ bool url_aggregator::set_hostname(const std::string_view input) {
   return components;
 }
 
+std::string ada::url_aggregator::to_string() const {
+  return components.to_string();
```
Comment (lemire) [inline src/url_aggregator.cpp:103]: Is that what you really want from to_string() in this case ?

(Honest question: I have no opinion.)

**Review by lemire — APPROVED** [review]
This has been annoying me from day one. So yeahhhh!

File: include/ada/implementation.h [hunk 2]
```diff
@@ -29,21 +29,13 @@ namespace ada {
    *
    * @param input the string input to analyze.
    * @param base_url the optional string input to use as a base url.
-   * @param encoding encoding (default to UTF-8)
```
Comment (lemire) [inline include/ada/implementation.h:32]: We may still want to document that we expect UTF-8. It is better to be explicit.

File: src/url_aggregator.cpp (hunk 1 above)
Comment (anonrig) [reply src/url_aggregator.cpp:103]: Well, I think this behaves like a `print()` or `debug_print()`. I don't think we need a to_string.

File: include/ada/implementation.h (hunk 2 above)
Comment (anonrig) [reply include/ada/implementation.h:32]: Just updated the description. Nice catch!

File: src/url_aggregator.cpp (hunk 1 above)
Comment (lemire) [reply src/url_aggregator.cpp:103]: I have no opinion, I was just raising the point for discussion.

**Pull request merged**